android/app/build.gradle

```diff
@@ -21,6 +21,21 @@ if (flutterVersionName == null) {
     flutterVersionName = '1.0'
 }
 
+def dartEnvironmentVariables = []
+if (project.hasProperty('dart-defines')) {
+    dartEnvironmentVariables = dartEnvironmentVariables + project.property('dart-defines')
+            .split(',')
+            .collectEntries { entry ->
+                def pair = URLDecoder.decode(entry).split('=')
+                [(pair.first()): pair.last()]
+            }
+}
+
+def secretsFile = rootProject.file('../auth/secrets.json')
+def secrets = new groovy.json.JsonSlurper().parseText(secretsFile.text)
+def googleMapsDevApiKey = dartEnvironmentVariables.androidMapsKeyProd ?: secrets.googleMapsAndroidApiKeyDev ?: ""
+def googleMapsProdApiKey = dartEnvironmentVariables.androidMapsKeyDev ?: secrets.googleMapsAndroidApiKeyProd ?: ""
```

Comment:
zastanawiam się nad tym rozdzieleniem na dev i prod, bo w na bitrise z tego co widzę zapodajemy klucz `androidMapsKey`, skąd wzięło się `androidMapsKeyProd` i `androidMapsKeyDev` ? czy w ogóle potrzebujemy to rozdzielać skoro klucz będzie zależał od wykonanego workflowu na bitrise?

Reply:
niedopatrzenie, już poprawiłem